source/_posts/2016-11-08-Vapor-setup.md

```diff
@@ -0,0 +1,311 @@
+---
+title: 6 simple steps to get your first Vapor JSON response
+date: 2016-11-08 12:42:00
+tags: vapor,setup
+authorId: olha
+categories:
+- Vapor
+---
+
+6 simple steps to setup your first Vapor JSON response. We will cover the initial setup and how to make both POST and GET with controllers and models. 
+
+This tutorial will assume that you already have installed [Vapor](https://vapor.github.io/documentation/getting-started/install-swift-3-macos.html) and [Xcode 8](https://developer.apple.com/xcode/) and are farmiliar with Swift 3.
+
+### 1. Creating your project
+
+Now open your terminal and cd into the place you want your project.
+
+Now run
+
+```javascript
+vapor new --ProjectName
+```
+
+Vapor will now ask you if you want to open the project. (Just go 'y')
+
+If you get 
+
+```jconsole
+ld: library not found for -lCLibreSSL for architecture x86_64
+clang: error: linker command failed with exit code 1 (use -v to see invocation)
+```
+
+Then just open terminal and run 
+```vapor build --clean```, after that run a ```vapor xcode``` that should do the trick.
+
+And hopefully you should be ready to rock and roll.
+
+### 2. Creating a model
+
+Open the terminal again, and run 
+```touch Source/App/Models/Car.swift```
```

Comment:
Shouldn't this be `Sources` instead of `Source`? I'm trying to follow the tutorial step by step, and in my xcode project there's a folder called Sources instead of Source.